regression/goto-instrument/inline_17/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+main.c
+--function-inline func1
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+func2\(\)
+--
+^warning: ignoring
```

Comment:
I think it's fine. ``--function-inline func1`` inlines all the calls that ``func1`` makes but the call to ``func1`` itself remains. I've also added a test now that ``func1`` is called.